Manager/TicketManager.php

```diff
@@ -205,9 +205,17 @@ public function getTicketListQuery(UserManagerInterface $userManager, $ticketSta
 
         if (\is_object($user)) {
             if (!$userManager->hasRole($user, TicketRole::ADMIN)) {
-                $query
-                    ->andWhere('t.userCreatedObject = :user')
-                    ->setParameter('user', $user);
+                // allow custom permissions from final user class
+                if (\is_object($user) && method_exists($user, 'addConditionForAccess')) {
+                    $qb = $user->addConditionForAccess($$qb);
```

Comment:
Looks like you should be using $query rather than $qb

Reply:
Of course, sorry. I don't understand how it could have work...

I updated my branch but I can't find how to update the PR (I usally work with gitlab).